attestation-agent/attester/src/tdx/mod.rs

```diff
@@ -2,14 +2,10 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 //
-
-use crate::utils::pad;
-
+use super::tsm_report::*;
 use super::Attester;
 use anyhow::*;
 use base64::Engine;
-use log::debug;
```

Comment:
Is this change related to the PR?

Reply:
it was needed when I had the tdx-attest-rs disabled but I can restore. OTOH, it makes the code consistent with other places where `log::` is used in this file.

Reply:
fixed